docs/cookbook/recipe_sonata_admin_without_user_bundle.rst

```diff
@@ -26,38 +26,133 @@ User Entity
 This represents your security user, you can read more about it
 `here <https://symfony.com/doc/5.4/security.html#a-create-your-user-class>`__::
 
-    namespace App\Entity;
+    <?php
```

Comment:
As you see the <?php tag wasn't needed and all the code was indented.
Please follow the same code style to have a successful build.

Reply:
ok, thank you for your feedback